lib/connector.js

```diff
@@ -27,9 +29,11 @@ Connector.prototype.getTypes = function() {
 
 /**
  * Get the default data type for ID
+ * @param prop Property definition
  * @returns {Function} The default type for ID
  */
-Connector.prototype.getDefaultIdType = function() {
+Connector.prototype.getDefaultIdType = function(prop) {
+  /*jshint unused:false */
```

Comment:
Ok, now I see why you set `unused:false` in the master `jshintrc` and it makes perfect sense to do so. Could you please remove these inline directives then, since they are not needed anymore?